packages/login/src/vaadin-lit-login-overlay.js

```diff
@@ -35,6 +35,8 @@ class LoginOverlay extends LoginOverlayMixin(ElementMixin(ThemableMixin(PolylitM
         .opened="${this.opened}"
         .title="${this.title}"
         .description="${this.description}"
+        role="dialog"
+        aria-label="${this.title}"
```

Comment:
The only issue I can see is that if you attach an element with `slot="title"`, then the `aria-label` will be out of sync and it will display the defaul `App name` if nothing is set in the `title` property.

![Screenshot 2024-08-30 at 14 06 19](https://github.com/user-attachments/assets/fbdbbb9c-da6e-4559-8bd1-567195f77b29)

Reply:
Good catch, thanks. I will add some logic to handle this.

Reply:
Added observer to set `aria-labelledby` and remove `aria-label` when the slotted title is provided.